sparse_strips/vello_cpu/src/dispatch/multi_threaded/worker.rs

```diff
@@ -177,7 +165,14 @@ impl Worker {
             }
         }
 
-        result_sender.send(task_idx as usize, task_buf).unwrap();
+        let strips_slice = self.strip_storage.strips.as_slice();
+
+        let task = CoarseTask {
+            strips: strips_slice.into(),
```

Comment:
There might be a similar way to preserve `self.strip_storage.strips` allocations too 🤔